File: Abstraction.py
class Parent:
    def __init__(self,name):
        self.name = name

    def Relation():
        pass

class Yeasin(Parent):
    def Relation(self):
        print(f"Yeasin parent is {self.name}")


class Rohim(Parent):
    def Relation(self):
        print(f"Rohims parent is {self.name}")

yeasin = Yeasin("Jalal")
yeasin.Relation()

rohim = Rohim('Rakib')
rohim.Relation()

File: Abstraction.cpp
#include <bits/stdc++.h>
using namespace std;

class Parent{
    public:
    string name;

    Parent(string name){
        this->name = name;
    }

    void Relation(string name){
        cout<<name;
    }
};

class Yeasin:public Parent{
    public:
    Yeasin(string name) : Parent(name) {}
    void Relation(){
        cout<<"Yeasin's parent is " << name << endl;
    }
};

int main()
{
    Yeasin yeasin("jalal");
    yeasin.Relation();
    return 0;
}